.changes/unreleased/charts-redpanda-Added-20250624-161712.yaml

```diff
@@ -0,0 +1,4 @@
+project: charts/redpanda
+kind: Added
+body: New `skip-cluster-configuration-sync` flag that disables cluster configuration in sidecar container, so that sidecar will not conflict with operator cluster configuration synchronization.
```

Comment:
Let's omit this from the changelog. This isn't a user facing flag. The fixed message in the operator log is more than enough.

Reply:
Removed